npg_qc_viewer/root/static/scripts/qcoutcomes/qc_outcomes_view.js

```diff
@@ -34,6 +34,38 @@ define([
 ) {
   var ID_PREFIX = 'rpt_key:';
 
+  /*
+   *This function assigns an image in the .lane column to signal the outcome of
+   *the usability flag from a certain tag.
+   *
+   *It requires a DOM object (obj) and a boolean flag (usability) displaying wether
+   *the usability of the tag lane is switched on or off. It also require the presence of the
+   *src images: bigTickBlue.png and bigCrossRed.png at the right path "/static/images/"
+   *
+   * Example:
+   *
+   *    $('.lane').each(function (i, obj) {
+   *     mqc_outcomes.usabilityDisplaySwitch(obj,1);
+   *   });
+   *
+   */
+  var usabilityDisplaySwitch = function (obj, usability) {
+    if (typeof obj === 'undefined') {
```

Comment:
Think is possible to improve validation

To make test more stringent
typeof obj !== 'object'
"obj must be defined and of type object"
To test its a DOM element
!obj instanceof Element
"obj must be defined and of type (DOM) Element"